awe_workbench/web/websocketClient.py

```diff
@@ -1,16 +1,24 @@
-#!/usr/bin/env python3.10
-# Copyright 2022, Educational Testing Service
+"""
+--- [ Test: startServers.py ] -----------------------------------------------------------
+
+Code for kicking off the parserServer.
+
+Author: Caleb Scott (cwscott3@ncsu.edu)
+
+Copyright 2022, Educational Testing Service
+
+-----------------------------------------------------------------------------------------
+"""
+
+# --- [ IMPORTS ] -----------------------------------------------------------------------
```

Comment:
I'm not really the biggest fan of these types of comments. I understand that it helps break up the code a little bit, but especially in smaller files, I don't see the need for them. Even in large files, the code already does this to some degree (all the imports say `import` or `from ... import`, all the classes start with `class`, etc.).